### PR TITLE
feat(one-app-router): update to 1.1.0

### PR DESCRIPTION
**Pull request by JAdshead**
Updating to version 1.1.0 of [one-app-router](https://github.com/americanexpress/one-app-router)

## Description
Updating to the latest version of one app router
## Motivation and Context
enables `matchPromise`, a proimisified version of the match api
removes UNSAFE_lifecycle methods from the router context. 

## How Has This Been Tested?
Unit and integration tests.

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?
Can make use of one-app-routers matchPromise

### CODE REVIEW

**Comment by JAdshead** [comment]
> So `RouterContext` was updated with `matchPromise`? We don't have to do anything from a Module Developer perspective to have errors go away?

This does not drop all `UNSAFE_*` errors but has removed a couple

File: __tests__/server/middleware/createRequestHtmlFragment.spec.js [hunk 1]
```diff
@@ -15,17 +15,24 @@
...
 // getBreaker is only added in the mock
 /* eslint-disable-next-line import/named */
 import { getBreaker } from '../../../src/server/utils/createCircuitBreaker';
 
 import * as reactRendering from '../../../src/server/utils/reactRendering';
 
-jest.mock('../../../src/universal/utils/matchPromisified');
+// const { browserHistory } = jest.requireActual('@americanexpress/one-app-router');
+
+// jest.mock('@americanexpress/one-app-router');
```
Comment (infoxicator) [inline __tests__/server/middleware/createRequestHtmlFragment.spec.js:29]: are these 2 comments intended here?

**Comment by JAdshead** [comment]
> Can you explain how did `matchPromisified` or `match` throw the `UNSAFE_*` console errors? I can't find the relationship, (but also I didn't know we had our own version of matchPromise in one-app either 😅)

https://github.com/americanexpress/one-app/pull/137#pullrequestreview-409908156

File: __tests__/server/middleware/createRequestHtmlFragment.spec.js (hunk 1 above)
Comment (JAdshead) [reply __tests__/server/middleware/createRequestHtmlFragment.spec.js]: fixed - 16dff60

**Pull request merged**